TrackletAlgorithm/MatchEngineUnit.h

```diff
@@ -93,73 +117,110 @@ inline void step(const VMStubMECM<VMSMEType> stubmem[4][1024]) {
   //If the buffer is not empty we have a projection that we need to 
   //process. 
   
-  NSTUBS istubtmp=istub_;
+  NSTUB istubtmp=istub_;
+  NSTUB iusetmp=iuse_;
```

Comment:
Why is iuse_ and iusetmp of type NSTUB? Don't we need only two bits for this?

Reply:
Switched to `ap_uint<2>`